infra/gravity/cluster_failover.go

```diff
@@ -0,0 +1,118 @@
+package gravity
```

Comment:
License header is missing.

Reply:
Should license header be included in every .go file?
Looks like all the files in in robotest are missing them.

Reply:
Yes, all source files should include the license header.